### DISCUSSION TITLE
Powerbank recommendation for Onoff Shim

### DISCUSSION THREAD

**Discussion by Sheepiely** [MiczFlor/RPi-Jukebox-RFID#1459] [Q&A] [answered]
I have an OnOff Shim installed on my Pi Zero W and use it together with a [Mi 10000mAh Power Bank 3](https://www.mi-store.co.nz/product/VXN4274GL/Mi-10000mAh-Power-Bank-3). The Onoff shim works great with shut down but it can't be used to turn on the Pi again after the powerbank has been turned off for more than 30s. I've tried different ports on the powerbank. I need to turn the power bank on first before using the Onoff shim switch to turn the Pi on.

I see that the [Easyacc 20000mAh Power bank](https://www.amazon.com/EasyAcc-20000mAh-Portable-Recharge-Capacity/dp/B01422TC14) has been recommended a lot but it is no longer available for sale.

Does anyone else have any consistent success with using a specific Powerbank with the Onoff shim?

Thanks.

**Comment by ThSGM** [comment]
I'm curious in this question as well. 

The guide here: https://www.thingiverse.com/thing:3789675

The author used both the OnOff shim and the powerbank here: https://www.amazon.de/Anker-PowerCore-Powerbank-VoltageBoost-Technologie/dp/B07HBWF1K8/

and did not seem to remark any unusual issues. However, the author seems largely non-responsive in the comment section. 

Another question I am interested in is how the OnOff shim differs from using a simple button and script: https://github.com/fire1ce/raspberry-pi-power-button#uninstall

**Reply by Sheepiely** [reply]
Without the onoff shim and just a simple on off button, when the pi is turned off, it is still drawing power. This means that in my case, the RFID reader connected to the pi is still on and draws power from the power bank and depletes it quicker. With an onoff shim, the power is completely cut.

**Reply by Sheepiely** [reply]
Got a recommendation from this post: https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/1180. Still waiting for the powerbank to be shipped. Will report back when I get it.

**Reply by s-martin** [reply]
If you speak German, please have a look here:

https://splittscheid.de/phoniebox-bauanleitung-toniebox-alternative/#ftoc-installation-onoffshim

**Comment by Sheepiely** [comment, marked as answer]
@ThSGM  I can confirm that the [ADATA PT100](https://www.adata.com/us/consumer/312) works perfectly with the Onoff Shim and the Pi Zero W.

**Comment by steveleader** [comment]
Interestingly I use the Mi Pro 3 20.000 mAh powerbank and OnOffShim without any issues.
This is the exact model I used https://www.amazon.de/dp/B07YCTZVZ9/
Only when connecting the charger, the usual voltage dip and therewith a reboot of the pi occurs.
